2019-03-29 11:35:15,694 ERROR 17420 [-/172.18.11.29/-/24ms POST /user/login] nodejs.TypeError: rule type must be one of number, int, integer, string, id, date, dateTime, datetime, boolean, bool, array, object, enum, email, password, url, but the following type was passed: phone
    at Parameter.validate (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/parameter/index.js:106:15)
    at Object.validate (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-validate/app/extend/context.js:12:39)
    at LoginController.index (/Users/yangfan/Desktop/yf/my-mall/mall-service/app/controller/user/login.js:21:26)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/router.js:195:33)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:56
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at i18n (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-i18n/app/middleware/i18n.js:13:12)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at dta (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/dta.js:12:12)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at xframe (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/xframe.js:7:11)
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 17420
hostname: yangfandeMacBook-Pro.local

2019-03-29 12:19:06,324 ERROR 11910 nodejs.ClusterClientNoResponseError: client no response in 1988810ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 11910
hostname: yangfandeMacBook-Pro.local

2019-03-29 13:24:41,507 ERROR 11910 nodejs.ClusterClientNoResponseError: client no response in 1914589ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 11910
hostname: yangfandeMacBook-Pro.local

2019-03-29 13:33:44,048 ERROR 11910 nodejs.ClusterClientNoResponseError: client no response in 502512ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 11910
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:58:23,620 ERROR 21841 [-/127.0.0.1/-/0ms GET /] nodejs.TypeError: Path must be a string. Received { fieldClass: 'rulesClass' }
    at assertPath (path.js:28:11)
    at Object.join (path.js:1236:7)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:14:44)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3

pid: 21841
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:58:24,876 ERROR 21842 [-/127.0.0.1/-/0ms GET /] nodejs.TypeError: Path must be a string. Received { fieldClass: 'rulesClass' }
    at assertPath (path.js:28:11)
    at Object.join (path.js:1236:7)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:14:44)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3

pid: 21842
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:58:47,169 ERROR 21843 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:14:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21843
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:58:48,301 ERROR 21844 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:14:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21844
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:59:03,115 ERROR 21849 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:14:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21849
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:59:04,275 ERROR 21854 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:14:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21854
hostname: yangfandeMacBook-Pro.local

2019-03-29 17:59:59,714 ERROR 21856 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21856
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:00:00,844 ERROR 21857 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21857
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:00:27,135 ERROR 21862 [-/127.0.0.1/-/1ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21862
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:00:28,254 ERROR 21863 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21863
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:13,118 ERROR 21868 [-/127.0.0.1/-/1ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21868
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:14,283 ERROR 21874 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21874
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:34,570 ERROR 21875 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21875
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:35,637 ERROR 21876 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21876
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:40,269 ERROR 21877 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21877
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:41,374 ERROR 21882 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21882
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:50,764 ERROR 21884 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21884
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:01:51,915 ERROR 21885 [-/127.0.0.1/-/0ms GET /] nodejs.AssertionError [ERR_ASSERTION]: options.property is required
    at new ContextLoader (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/context_loader.js:49:5)
    at AppWorkerLoader.loadToContext (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/egg_loader.js:408:5)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
generatedMessage: false
name: "AssertionError [ERR_ASSERTION]"
code: "ERR_ASSERTION"
actual: undefined
expected: true
operator: "=="
pid: 21885
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:03:15,412 ERROR 21887 [-/127.0.0.1/-/0ms GET /] nodejs.TypeError: Path must be a string. Received { fieldClass: 'rulesClasses' }
    at assertPath (path.js:28:11)
    at Object.join (path.js:1236:7)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:44)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3

pid: 21887
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:03:16,555 ERROR 21888 [-/127.0.0.1/-/1ms GET /] nodejs.TypeError: Path must be a string. Received { fieldClass: 'rulesClasses' }
    at assertPath (path.js:28:11)
    at Object.join (path.js:1236:7)
    at AppBootHook.didLoad (/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js:15:44)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:42)
    at process.nextTick (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/lifecycle.js:248:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3

pid: 21888
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:08:05,060 ERROR 22000 [-/172.18.11.29/-/25ms POST /user/login] nodejs.TypeError: ctx.rules is not a function
    at LoginController.index (/Users/yangfan/Desktop/yf/my-mall/mall-service/app/controller/user/login.js:23:25)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/router.js:195:33)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:56
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at i18n (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-i18n/app/middleware/i18n.js:13:12)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at dta (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/dta.js:12:12)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at xframe (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/xframe.js:7:11)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at xssProtection (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 22000
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:08:19,352 ERROR 22007 [-/172.18.11.29/-/23ms POST /user/login] nodejs.TypeError: ctx.rules is not a constructor
    at LoginController.index (/Users/yangfan/Desktop/yf/my-mall/mall-service/app/controller/user/login.js:23:21)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/@eggjs/router/lib/router.js:195:33)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at /Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:56
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at i18n (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-i18n/app/middleware/i18n.js:13:12)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at dta (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/dta.js:12:12)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at xframe (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/xframe.js:7:11)
    at dispatch (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/koa-compose/index.js:42:32)
    at xssProtection (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-security/lib/middlewares/xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 22007
hostname: yangfandeMacBook-Pro.local

2019-03-29 18:35:53,755 ERROR 20067 nodejs.unhandledExceptionError: ENOENT: no such file or directory, stat '/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js'
    at Object.fs.statSync (fs.js:948:11)
    at DevelopmentEventSource._onFsWatchChange (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/egg-watcher/lib/event-sources/development.js:82:16)
    at emitTwo (events.js:126:13)
    at FSWatcher.emit (events.js:214:7)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1364:12)
errno: -2
code: "ENOENT"
syscall: "stat"
path: "/Users/yangfan/Desktop/yf/my-mall/mall-service/app.js"
name: "unhandledExceptionError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-29 20:07:48,291 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 127622ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-29 20:47:34,419 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 2366118ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-29 23:46:51,699 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 2054824ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-30 01:47:49,861 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 7218147ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-30 03:48:51,461 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 7221583ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-30 05:49:29,604 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 7218106ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-30 07:50:48,606 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 7218978ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-30 09:51:27,636 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 7219013ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

2019-03-30 10:10:52,977 ERROR 20067 nodejs.ClusterClientNoResponseError: client no response in 1125318ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yangfan/Desktop/yf/my-mall/mall-service/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 20067
hostname: yangfandeMacBook-Pro.local

